spec/presenters/world_index_presenter_spec.rb

```diff
@@ -0,0 +1,14 @@
+RSpec.describe WorldIndexPresenter do
+  let(:fixture) { fetch_fixture("world_index") }
```

Comment:
Are we able to load the example of content schemas in Publishing API?  That would save having such a large JSON file here.

Reply:
Done that, thank you!